build/replace_maven-wagon-http-version.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -xe
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# patches installed maven version to get fix for https://issues.apache.org/jira/browse/HTTPCORE-634
+
+MAVEN_HOME=$(mvn -v |grep 'Maven home:' | awk '{ print $3 }')
+if [ -d "$MAVEN_HOME" ]; then
+  cd "$MAVEN_HOME/lib"
+  rm wagon-http-*-shaded.jar
+  # built from https://github.com/lhotari/maven-wagon/commits/5ff79d284
+  curl -O https://jitpack.io/com/github/lhotari/maven-wagon/wagon-http/5ff79d284/wagon-http-5ff79d284-shaded.jar
```

Comment:
This change won't be easily accepted.
No one knows what is inside your jar

Reply:
I agree, but this has to be resolved in a way or another. there's a PR https://github.com/apache/maven-wagon/pull/76 to get this to upstream. Please review it.